src/client_side_request.cc

```diff
@@ -1794,9 +1794,14 @@ ClientHttpRequest::doCallouts()
             ACLFilledChecklist ch(NULL, request, NULL);
             ch.src_addr = request->client_addr;
             ch.my_addr = request->my_addr;
-            nfmark_t mark = aclMapNfmark(Ip::Qos::TheConfig.nfmarkToClient, &ch);
-            if (mark)
+            acl_nfmark *l = aclMapNfmark(Ip::Qos::TheConfig.nfmarkToClient, &ch);
+            if (l) {
```

Comment:
Please merge the above two lines together to avoid exposing the rest of the code to (a possibly nil) `l`:

```
if (const auto mc = aclMapNfmark(Ip::Qos::TheConfig.nfmarkToClient, &ch))
```